frontend/test/metabase/scenarios/dashboard/subscriptions.cy.spec.js

```diff
@@ -0,0 +1,47 @@
+import { restore, signInAsAdmin, USERS } from "__support__/cypress";
+const { admin } = USERS;
+
+describe("scenarios > dashboard", () => {
+  beforeEach(() => {
+    restore();
+    signInAsAdmin();
+    // Dummy SMTP setup
+    cy.request("PUT", "/api/setting", {
+      "email-smtp-host": "smtp.foo.test",
+      "email-smtp-port": "587",
+      "email-smtp-security": "none",
+      "email-smtp-username": "nevermind",
+      "email-smtp-password": "it-is-secret-NOT",
+      "email-from-address": "nonexisting@metabase.test",
+    });
+  });
+
+  it.skip("should persist attachments for dashboard subscriptions (metabase#14117)", () => {
+    // Orders in a dashboard
+    cy.visit("/dashboard/1");
+    cy.get(".Icon-share").click();
+    cy.findByText("Dashboard subscriptions").click();
+    cy.findByText("Email it").click();
+    cy.findByPlaceholderText("Enter user names or email addresses")
+      .click()
+      .type(`${admin.first_name} ${admin.last_name}{enter}`);
+    // This is extremely fragile
+    // TODO: update test once changes from `https://github.com/metabase/metabase/pull/14121` are merged into `master`
+    cy.findByText("Attach results")
+      .parent()
+      .parent()
+      .next()
+      .find("a") // Toggle
+      .click();
+    cy.findByText("Questions to attach").click();
+    cy.contains("Done")
+      .closest(".Button")
+      .should("not.be.disabled")
+      .click();
+    cy.findByText("Subscriptions");
+    cy.findByText("Emailed daily at 8:00 AM").click();
+    cy.findByText("Delete this subscription").scrollIntoView();
+    cy.findByText("Questions to attach");
+    cy.findByText("Orders");
```

Comment:
Which cards are on this dashboard? And wouldn't `cy.findByText("Orders");` also detect the card title?
I think a correct assertion would be the check the toggle and checkbox.

Reply:
Spot on, as always! I appreciate your reviews so much, @flamber ! Thank you.
The card's title is `Orders` so yes - it would've made a conflict because there are 2 strings with that name. 

I've updated the test with your suggestions. To make sure everything works, I've commented out this (only locally) to see how the test plays out:
```javascript
cy.findByText("Attach results")
  .parent()
  .parent()
  .next()
  .find("a") // Toggle
  .click();
cy.findByText("Questions to attach").click();
// cy.contains("Done")
//   .closest(".Button")
//   .should("not.be.disabled")
//   .click();
// cy.findByText("Subscriptions");
// cy.findByText("Emailed daily at 8:00 AM").click();
// cy.findByText("Delete this subscription").scrollIntoView();
// cy.findByText("Questions to attach");
cy.findAllByRole("listitem")
  .contains("Orders") // yields the whole <li> element
  .within(() => {
    cy.findByRole("checkbox").should("have.attr", "aria-checked", "true");
  });
```

![image](https://user-images.githubusercontent.com/31325167/103136097-bbe45a00-46bd-11eb-93ba-376270d18cb1.png)

It works as expected, so I'm going to commit the updated test now.